docs/source/format/index.rst

```diff
@@ -15,10 +15,13 @@
 .. specific language governing permissions and limitations
 .. under the License.
 
+.. _format:
+
+Specifications and Protocols
+============================
+
 .. toctree::
    :maxdepth: 2
-   :caption: Specifications and Protocols
```

Comment:
Out of curiosity, what happens if you keep this caption? Does it then appear in the sidebar (between the standard "Section Navigation" and the actual content)

Reply:
Yes, and also in the toctree:

![Screenshot 2023-08-31 at 12 33 40](https://github.com/apache/arrow/assets/16418547/4dd2062e-2a1a-4ea2-b0f4-ecec569a577c)

vs.

![Screenshot 2023-08-31 at 12 37 44](https://github.com/apache/arrow/assets/16418547/bde68f19-6f15-4d7a-bf9f-860c50b53ef8)

Reply:
OK, that's a good change then ;)